fs/layer/operations/ops.go

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
It might be worth just adding `ops.go` to layer instead of creating a new package